### DISCUSSION TITLE
Sur Ma Cantine en général

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1031] [Retours utilisateur·rice]
08/02, retour utilisateur (un réseau bio qui livre en produits bios les cantines de la région Centre – Val de Loire) : "outil extrêmement intéressant", "formidable"

**Comment by alemangui** [comment]
Merci pour ce retour :heart:  (D'ailleurs je me suis permis d'enlever les infos personnels :) )

**Reply by ghost** [reply]
Pas de souci. Je t'en prie!

**Comment by ghost** [comment]
10/02, L. A. (Bio Centre) : "outil facile, intuitif et adapté aux collectivités"
"outil puissant car compilation d'informations et possibilité de mise en relation"

**Comment by ghost** [comment]
11/02, B. L. C. : "hyper intéressant", "c'est ce dont on rêvait"

**Comment by ghost** [comment]
18/02, A. E. : "le site est hyper bien fait"

**Comment by ghost** [comment]
23/02, P. G. : "super outil"
